### PR TITLE
Gaming Anthems

### PR DESCRIPTION
**Pull request by techlewi**
image

### CODE REVIEW

**Comment by techlewi** [comment]
> Please add more context to your presence, it being that way is not different than a regular status.

what do you mean by that?

**Comment by doomlerd** [comment]
> what do you mean by that?
The status never changes, look at the code of other presences to get a better idea.

**Comment by techlewi** [comment]
> > what do you mean by that?
> > The status never changes, look at the code of other presences to get a better idea.

I'm not that good at java coulda dev help me?

**Comment by techlewi** [comment]
dont worry ill do it later!

**Comment by doomlerd** [comment]
3 days to close the PR, you should do so before or you can re-open another PR later.

File: Gaming Anthems/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "TechLewi",
+    "id": "590575157718941708"
+  },
+  "service": "GamingAnthems",
```
Comment (Alanexei) [inline Gaming Anthems/dist/metadata.json:6]: Change the name of this to `Gaming Anthems`, the directory name of the presence must match.

**Pull request closed without merging**